corehq/apps/userreports/util.py

```diff
@@ -201,12 +202,14 @@ def get_static_report_mapping(from_domain, to_domain, report_map):
                 ''
             )
             is_custom_report = False
-        else:
+        elif static_report.get_id.startswith(CUSTOM_REPORT_PREFIX):
             report_id = static_report.get_id.replace(
                 CUSTOM_REPORT_PREFIX + from_domain + '-',
                 ''
             )
             is_custom_report = True
+        else:
+            raise UserReportsError("Found dynamic UCR when static expected")
```

Comment:
I don't think this will ever get raised since this loop is only iterating over static UCRs.